
Lap_Count: 1
Water_Reservoir_Location_Detected: 0
Water_Discharge_Location_Detected: 0
Water_Sucked_by_Syringes : 0
Water_Released_by_Syringes : 0
Water_Reservoir_Location_Latitude : 0.0
Water_Reservoir_Location_Longitude : 0.0
Water_Reservoir_Location_Altitude : 0.0
Water_Discharge_Location_Latitude : 0.0
Water_Discharge_Location_Longitude : 0.0
Water_Discharge_Location_Altitude : 0.0